test/shared/backgroundScripts/contextMenus.test.js

```diff
@@ -3,13 +3,17 @@ import { describe, it } from "mocha";
 
 import { initContextMenu } from "../../../build/chromium/shared/backgroundScripts/contextMenus.js";
 
-import { setStorage, getLocaleMessage, setExtensionIsChromium } from "../../setup.test.js";
+import {
+  setStorage,
+  getLocaleMessage,
+  setExtensionIsFirefoxExtension,
+} from "../../setup.test.js";
 
 describe("shared/backgroundScripts/contextMenus.js", () => {
   describe("initContextMenu()", () => {
     it("should make the shared context menu", async () => {
       setStorage("currentExternalBrowser", "Firefox");
-      setExtensionIsChromium(true);
+      setExtensionIsFirefoxExtension(true);
```

Comment:
FYI: This method is doing the opposite of what the line was doing before.

Reply:
The tests are currently not working, and this method would not correctly replace the new globals defined by webpack. I'm having a bit of trouble making the tests work, but I will update everything accordingly. 

I won't be merging until the tests have been updated to reflect the new build process

Reply:
I have made a PR to this branch for fixing the tests: https://github.com/mozilla/firefox-launch/pull/26

Let me know if this can merge as you have stated you did not previously review unit tests

Reply:
No objections to merging.